app/code/Magento/CmsGraphQl/Model/Resolver/DataProvider/Page.php

```diff
@@ -44,10 +62,32 @@ public function __construct(
      * @return array
      * @throws NoSuchEntityException
      */
-    public function getData(int $pageId): array
```

Comment:
Public method was removed - it is forbidden.
Check this article: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/
**section** "Public and protected method removal"
Please mark `getData` method as a _@depricated_ instead of removing it.

Reply:
I've created a new data provider model instead of make changes in the current one. 

Could you please review?

Thank you, @vpodorozh !